src/totp/converters.ts

```diff
@@ -0,0 +1,16 @@
+export const hexToUint8Array = (hex: string): Uint8Array => {
+  const length = hex.length / 2;
+  const uintArray = new Uint8Array(length);
+  for (let i = 0; i < length; i++) {
+    uintArray[i] = parseInt(hex.substring(2 * i, 2 * i + 2), 16);

```

Comment:
I would expect some comments or putting these numbers into constants for the logic to be more explanitory

Reply:
ℹ️ I've added a comment and made a local variable`hexPair`.